tests/test_jump.py

```diff
@@ -31,18 +32,6 @@ def _cube(ngroups, readnoise=10):
     return _cube
 
 
-@pytest.mark.skipif(not OPENCV_INSTALLED, reason="`opencv-python` not installed")
-def test_find_simple_circle():
-    plane = np.zeros(shape=(5, 5), dtype=np.uint8)
-    plane[2, 2] = DQFLAGS['JUMP_DET']
-    plane[3, 2] = DQFLAGS['JUMP_DET']
-    plane[1, 2] = DQFLAGS['JUMP_DET']
-    plane[2, 3] = DQFLAGS['JUMP_DET']
-    plane[2, 1] = DQFLAGS['JUMP_DET']
-    circle = find_circles(plane, DQFLAGS['JUMP_DET'], 1)
-    assert circle[0][1] == pytest.approx(1.0, 1e-3)
-
-
 @pytest.mark.skipif(not OPENCV_INSTALLED, reason="`opencv-python` not installed")
```

Comment:
Could probably remove this line too, now that opencv is installed by default.

Reply:
removed